src/main/config/wikipedia-application-local-runner.properties

```diff
@@ -26,7 +26,7 @@ app.processor-id-generator.class=org.apache.samza.runtime.UUIDGenerator
 task.name.grouper.factory=org.apache.samza.container.grouper.task.GroupByContainerIdsFactory
 
 # Serializers
-serializers.registry.json.class=org.apache.samza.serializers.JsonSerdeFactgory
```

Comment:
Can we remove `UUIDGenerator` since it's not a mandatory config? 
Does the program run without it?